link/src/main/java/com/meshconnect/link/ui/LinkActivity.kt

```diff
@@ -171,6 +172,7 @@ internal class LinkActivity : AppCompatActivity() {
             webViewClient = WebClient()
             webChromeClient = ChromeClient()
             loadUrl(url)
+            evaluateJavascript(meshSDKPlatformScript, null)
```

Comment:
can we place this line before `loadUrl` this way, we will make sure the `window` object has values after the page loads (for the backend)